### PR TITLE
lp-v2: fix message fields

### PR DESCRIPTION
**Pull request by wischli**
# Description

* Fixes LP v2 message fields
* Bumps spec versions to v0.13.3 which I needed for Dev deployments (currently on v0.13.3 already)
* Fixes passthrough adapter code introduced in https://github.com/centrifuge/centrifuge-chain/pull/1888
* Includes https://github.com/centrifuge/centrifuge-chain/pull/1929 which was required on Dev

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: pallets/liquidity-pools/src/message.rs
Comment (lemunozm) [inline pallets/liquidity-pools/src/message.rs:235]: Could you update the specs with this change? 🙏🏻 (or if there is a maintainer, the maintainer)

**Review by lemunozm — APPROVED** [review]
LGTM! although not sure if this is ready to review. 

Q: From where the message changes comes from?

File: pallets/liquidity-pools/src/message.rs
Comment (wischli) [reply pallets/liquidity-pools/src/message.rs:235]: This change was a result from syncing with the spec: https://centrifuge.hackmd.io/Soy4JiFbRSyovj53KW-a4A?view#DisputeMessageRecovery

File: runtime/integration-tests/src/cases/lp/transfers.rs
Comment (lemunozm) [inline runtime/integration-tests/src/cases/lp/transfers.rs:147]: Working then? 🙌🏻

File: pallets/liquidity-pools/src/message.rs
Comment (lemunozm) [reply pallets/liquidity-pools/src/message.rs:235]: Ok! Somehow my hackmd was not updated with the latest version of that document

**Review by lemunozm — APPROVED** [review]
Not sure if done, but LGTM as it is!

File: runtime/integration-tests/src/cases/lp/transfers.rs
Comment (lemunozm) [inline runtime/integration-tests/src/cases/lp/transfers.rs:313]: @wischli there is a fourth ignored test in this file in line `238`, is it still failing? or should have been fixed with the latest solidity update?

Comment (lemunozm) [reply runtime/integration-tests/src/cases/lp/transfers.rs:313]: If so, we should remove the "un-ignore" it

Comment (wischli) [reply runtime/integration-tests/src/cases/lp/transfers.rs:313]: All ignored tests currently fail unfortunately.

Comment (lemunozm) [reply runtime/integration-tests/src/cases/lp/transfers.rs:313]: But I'm not referring to the investment ones, just another from `transfers`

**Pull request merged**
